### PR TITLE
Fixed issues for  #181

### PR DESCRIPTION
**Pull request by bobbycrooz**
Fixed issues for #181 [issue link here](https://github.com/zurichat/zc_plugin_company_goals/issues/181)
Implemented dropdown to show list of options to sort Goals by
[view live link here: ](https://goals.zuri.chat) on live link click sort button

### CODE REVIEW

File: client/src/components/goal interface navbar/NavLayout.js [hunk 1]
```diff
@@ -29,14 +29,65 @@ const Sort = styled.div`
...
 const NavLayout = () => {
+  function showDrpDw() {
+    const sortDrpContainer = document.getElementById('sort_drop_down');
+    sortDrpContainer.classList.toggle('active');
+  }
+
   return (
     <GridLayout>
-      <NavName className="active"> all goals </NavName> <NavName> annual goals </NavName>
+      <NavName className=""> all goals </NavName> <NavName> annual goals </NavName>
```
Comment (abiona01) [inline client/src/components/goal interface navbar/NavLayout.js:77]: Your className is empty, kindly remove or add a valid className

**Review by abiona01 — CHANGES_REQUESTED** [review]

File: client/src/index.css [hunk 2]
```diff
@@ -10,6 +10,7 @@ body
 
 body {
   overflow-x: hidden;
+  color: #95959548,;
 }
 
 body {
```
Comment (abiona01) [inline client/src/index.css:16]: Did you create this index.css file? You're supposed to style with styled components.

**Review by abiona01 — CHANGES_REQUESTED** [review]

**Review by abiona01 — APPROVED** [review]

File: client/src/index.css (hunk 2 above)
Comment (bobbycrooz) [reply client/src/index.css]: it removed already

File: client/src/components/goal interface navbar/NavLayout.js (hunk 1 above)
Comment (bobbycrooz) [reply client/src/components/goal interface navbar/NavLayout.js]: fixed

**Review by abiona01 — APPROVED** [review]

File: client/src/index.css (hunk 2 above)
Comment (abiona01) [reply client/src/index.css]: Alright

**Pull request merged**